app/src/main/java/io/github/iurimenin/upcomingmovies/model/MovieVO.kt

```diff
@@ -8,23 +8,38 @@ import java.util.*
 /**
  * Created by Iuri Menin on 26/08/17.
  */
-class MovieVO : Parcelable {
+class MovieVO protected constructor(`in`: Parcel) : Parcelable {
 
-    var vote_count: String = ""
+    var vote_count: String? = ""
     var id: Int = 0
-    var vote_average: String= ""
-    var title: String = ""
-    var popularity: String = ""
-    var poster_path: String = ""
-    var original_language: String = ""
-    var original_title: String = ""
-    var backdrop_path: String = ""
-    var adult: String = ""
-    var overview: String = ""
-    var release_date: String = ""
+    var vote_average:  String? = ""
```

Comment:
Why are you not using camel case ?

Reply:
Because API returns a JSON with this format, so it was faster to convert this JSON to object using Gson